doc/api/corepack.md

```diff
@@ -96,31 +96,35 @@ it can conflict with such environments. To avoid that happening, call the
 the same time you're preparing your deploy image). This will ensure that the
 required package managers are available even without network access.
 
-The `pack` command has [various flags][]. Consult the detailed
+The `corepack` command has [various flags][]. Consult the detailed
 [Corepack documentation][] for more information.
 
 ## Supported package managers
 
 The following binaries are provided through Corepack:
 
-| Package manager | Binary names      |
-| --------------- | ----------------- |
-| [Yarn][]        | `yarn`, `yarnpkg` |
-| [pnpm][]        | `pnpm`, `pnpx`    |
+| Package manager  | Binary names      |
+| ---------------- | ----------------- |
+| [Yarn][]         | `yarn`, `yarnpkg` |
+| [pnpm][]         | `pnpm`, `pnpx`    |
+| [npm][]*         | `npm`, `npx`      |
+
+\* Not enabled by default. Use `corepack enable npm` to do so.
```

Comment:
If this is only referring to NPM, couldn't it be referenced by name, and not a "*"?

Reply:
I could do that if you want me to.
Wanted to raise attention that it is a special case as soon as you read the entry in the table.

Reply:
It’s your call, I’m not a CODEOWNER, so it’s up to you :-).